README.md

```diff
@@ -1,2 +1,275 @@
+
+<!-- - **Mobile Friendly** : this website performs well on a mobile device like a phone or tablet. -->
+<!-- - **Desktop Friendly**: this website is responsive to desktop screen sizes.
+- **Simple and Attractive Design** : the UI / UX is engaging and appealing.
+- **Functional** : this website is accessible, easy to navigate, and helpful for users. -->
+<!-- - **Animations and transitions : This website has engaging transitions and animations that makes it easier to use, smoother and more natural. -->
+<!-- - ** PopUp windows **: this features add to see the project details about each project in popup windows.
+-  ** Form validation **: this features validate the email should be lowercase for portfolio contact. -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> View the live demo of this project by clicking the text below.
+<!-- - [See Live Demo via Github pages](https://geekelo.github.io/bookstore-cms/) -->
+- [See Live Demo via Render](https://crypto-price-ranker-63gw.onrender.com/)
+- [First Presentation Video](https://youtu.be/1zCEFFYB-So)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+> Clone the repository
+```
+git clone https://github.com/geekelo/crypto-price-rank.git
+```
+> And you are ready to begin your project
+
+### Setup <a name="setup"></a>
+This project contains
+An HTML FILE (free to edit) - file that contains html codes to give structure to the main webpage
+A CSS FILE (free to edit) - file that contains css codes to style the webpage
+A GIT IGNORE FILE (free to edit) - to hide personal or private files
+HTML/CSS LINTER FILES (should not edit) - Do not make changes
+A JAVASCRIPT FILE (free to edit) - that handles functionalities and dynamic HTML
+
+### Prerequisites <a name="prerequisites"></a>
+> You should have Node and Git Installed
+> You should have basic knowledge on HTML / CSS / JavaScript
+> You should have a code editor
+
+### Install <a name="install"></a>
+
+> Download install [VSCODE](https://code.visualstudio.com/) and [Git](https://git-scm.com/)
+> To install linters, execute the following commands:
+```
+Initialize npm | ``` npminit -y ```
+HTML | ``` npm install --save-dev hint@7.x ```
+CSS | ```npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x  ```
+```
+### Usage <a name="usage"></a>
+To run the linters, execute the following command and fix linter errors:
+```
+HTML | ``` npx hint . ```
+CSS | ```npx stylelint "**/*.{css,scss}" ```
+If you get a flood of errors keep in mind that linters guide you in writing a clean code!
+```
+### Run Tests <a name="run-tests"></a>
+You can run this program on your browser
+
```

Comment:
- [ ] Please update this section to include the command that the user will need to run the tests in your project, such as `npm test`. 😊

Reply:
Hello @ridwanediallo 
Thank you for the review
I'd appreciate it if you could do me a favour by star-gazing this repository
Thanks in advance 😊

Reply:
Okay, @geekelo with pleasure. 😊